android/Volunteers-welcome!.md

```diff
@@ -5,7 +5,7 @@ We are always looking for volunteers, feel free to step in! It's very easy to ge
 # Developers
 
 1. Set up your working environment by following the [quick start guide](Quick-start-guide-for-Developers.md).
-2. Choose an [unassigned bug](https://github.com/commons-app/apps-android-commons/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee+-label%3Adebated+label%3Abug+-label%3A%22low+priority%22+-label%3Aupstream) that sounds interesting to you, and ask to be assigned the issue. If the issue was assigned to someone who has been inactive for over 1 month, you can bring this to the attention of collaborators who may ping them for updates and possibly re-assign the issue to you.
```

Comment:
How about retaining the list of excluded labels? It will help direct the contributors towards issues with higher impact. The current URL excludes low priority or debated issues, or the ones related to upstream problems.